wave_utils/tests/test_waves.py

```diff
@@ -8,28 +8,43 @@
 from __future__ import absolute_import, division, print_function, unicode_literals
 
 import numpy as np
+import pytest
 
-from wave_utils import waves 
+from wave_utils import waves
 
-#some constants:
-g = 9.806
-pi = np.pi
 
 def test_import():
     """ is there anything there? """
     assert hasattr(waves, 'wave_number')
 
+def test_frequency():
+    """
+    Test the wave frequency
+
+    The test is done for multiple values and most importantly edge cases, due to
+    the sqrt in the equation.
+    """
+    assert np.isclose(4.59043034, waves.frequency(waves.g, 3.0, 0.3), atol=1e-7)
```

Comment:
generally better to use the relative tolerance -- but these are all about the same magnitude, so this is OK.